### ISSUE TITLE
I assigned num_Graph=45 and it gave an error

### ISSUE THREAD

**Issue by sahinemreaslan** [liuxz1011/TodyNet#12]
You can try to repro this exception using the following code snippet. If that doesn't trigger the error, please include your original repro script when reporting this issue.

import torch
torch.backends.cuda.matmul.allow_tf32 = True
torch.backends.cudnn.benchmark = True
torch.backends.cudnn.deterministic = False
torch.backends.cudnn.allow_tf32 = True
data = torch.randn([16, 27, 256, 45], dtype=torch.float, device='cuda', requires_grad=True)
net = torch.nn.Conv2d(27, 18, kernel_size=[1, 3], padding=[0, 1], stride=[1, 1], dilation=[1, 1], groups=1)
net = net.cuda().float()
out = net(data)
out.backward(torch.randn_like(out))
torch.cuda.synchronize()

ConvolutionParams 
    data_type = CUDNN_DATA_FLOAT
    padding = [0, 1, 0]
    stride = [1, 1, 0]
    dilation = [1, 1, 0]
    groups = 1
    deterministic = false
    allow_tf32 = true
input: TensorDescriptor 0x148672820
    type = CUDNN_DATA_FLOAT
    nbDims = 4
    dimA = 16, 27, 256, 45, 
    strideA = 311040, 11520, 45, 1, 
output: TensorDescriptor 0x7fd4640288f0
    type = CUDNN_DATA_FLOAT
    nbDims = 4
    dimA = 16, 18, 256, 45, 
    strideA = 207360, 11520, 45, 1, 
weight: FilterDescriptor 0x7fd46401df30
    type = CUDNN_DATA_FLOAT
    tensor_format = CUDNN_TENSOR_NCHW
    nbDims = 4
    dimA = 18, 27, 1, 3, 
Pointer addresses: 
    input: 0x7fd4d4668000
    output: 0x7fd4d5964000
    weight: 0x7fd56111a800
